### DISCUSSION TITLE
Use with Astro

### DISCUSSION THREAD

**Discussion by basarozcan** [spatie/ray#866] [Q&A] [answered]
Is there any integration document or path to follow when integrate Ray for Astro?

**Comment by vivgui** [comment]
I would like to know this as well!

**Comment by vivgui** [comment, marked as answer]
I figured it out! You need to use the [node-ray](https://github.com/permafrost-dev/node-ray) package and use the `toJson` method, Here's an example of running it from an Astro file frontmatter:

``` js
---
import { ray } from 'node-ray';
let logos = (await getLogos()) || [];

ray().toJson(logos);
---
```

This returns the logos object all expanded:
![CleanShot 2024-06-20 at 16 58 45](https://github.com/spatie/ray/assets/7389358/d43c2bc2-acfb-475b-9bb4-57f344984948)
